02 - Community Expansions/02.01 Plugins by Category/Plugins to export markdown content.md

```diff
@@ -14,6 +14,7 @@ Plugins to convert from [[Markdown]].
 ## Plugins in this category
 
 - [[obsidian-export-to-tex|Export To TeX]]: Export vault files in a format amenable to pasting into a tex document
+- [[Straightforward-Obsidian-to-Latex]]: Exports complicated Obsidian notes to LateX. It respects embedded notes. Written in Python.
```

Comment:
This is fine.

There is also a 'Mathjax and LaTeX Plugins' category that might be appropriate, as an extra link.